cmake/toolchains/mt.runtime.win32.toolchain.cmake

```diff
@@ -29,6 +29,12 @@ if(use_static_runtime)
             string(REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
         endforeach()
     endforeach()
+    foreach(type EXE SHARED MODULE)
```

Comment:
let's add link to docs https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/using-the-microsoft-c-runtime-with-user-mode-drivers-and-apps

do we also need to patch flags for `STATIC` ?